### DISCUSSION TITLE
The correct Permission of Pterodactyl panel files

### DISCUSSION THREAD

**Discussion by csgo-s** [pterodactyl/panel#2959] [General Community]
Hi
What is the correct Permission of files and folders in these directories?
/www/pterodactyl
and also
/etc/pterodactyl

**Comment by trenutoo** [comment]
You configure the correct permission for all files by following the documentation [here](https://pterodactyl.io/panel/1.0/getting_started.html#set-permissions) and for wings [here](https://pterodactyl.io/wings/1.0/installing.html#installing-wings-2).
